awesome_dashboard/static/src/dashboard/dashboard.js

```diff
@@ -0,0 +1,93 @@
+import { Component, useState, onWillStart } from "@odoo/owl";
+import { registry } from "@web/core/registry";
+import { Layout } from "@web/search/layout";
+import { useService } from "@web/core/utils/hooks";
+import { DashboardItem } from "./dashboard_item";
+import { PieChart } from "./piechart/piechart";
+import { DashboardConfiguration } from "./dashboard_configuration/dashboard_configuration";
+
+
+class AwesomeDashboard extends Component {
+    static template = "awesome_dashboard.AwesomeDashboard";
+    static components = { Layout, DashboardItem, PieChart };
+
+    setup() {
+        this.action = useService("action");
+        this.dialog = useService("dialog");
+        this.orm = useService("orm");
+        this.statisticsService = useState(useService("awesome_dashboard.statistics"));
+
+        this.allItems = registry.category("awesome_dashboard").getAll();
+        this.state = useState({
+            hiddenItems: []
+        });
+
+        onWillStart(async () => {
+            await this.loadHiddenItems();
+        });
+    }
+
+    get items() {
+        const visibleItems = {};
+        Object.entries(this.allItems).forEach(([itemId, item]) => {
+            if (!this.state.hiddenItems.includes(item.backend_attribute)) {
+                visibleItems[itemId] = item;
+            }
+        });
+        return visibleItems;
+    }
+
+    async loadHiddenItems() {
+        try {
+            const config = await this.orm.call('res.users', 'get_dashboard_config', []);
+            const hiddenItems = [];
+            for (const [itemId, isVisible] of Object.entries(config)) {
+                if (!isVisible) {
+                    hiddenItems.push(itemId);
+                }
+            }
+            this.state.hiddenItems = hiddenItems;
```

Comment:
I'd use an array instead of a dict (it's easier to handle, at list it's more readable) but it's only the tutorial so it's ok.
But this part could be more "modern" like 
```suggestion
            this.state.hiddenItems = Object.entries(config)
			  .filter(([itemId, isVisible]) => !isVisible)
			  .map(([itemId]) => itemId);
```